### PR TITLE
[14.0][FIX] cetmix_tower_server: Security restrictions on the command wizard path field

### PR DESCRIPTION
**Pull request by Aldeigja**
Fixing an [issue](https://github.com/cetmix/cetmix-tower/pull/59#pullrequestreview-2364938160) reported in #59 
Before this PR:

Group_user members can't run command wizards on servers due to field "path" restricitions

After this PR:

Members of group_user can run command wizards on servers

### CODE REVIEW

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]
Please check the suggestions. Probably they will work faster.
Regarding tests, we need to ensure that commands run using wizard will be executed with correct path based on access group. So we need to check the execution results (command logs) for that.

**Comment by CetmixGitDrone** [comment]
Congratulations, PR rebased to [14.0-dev](https://github.com/cetmix/cetmix-tower/commits/14.0-dev).

**Comment by ivs-cetmix** [comment]
Hi @norlinhenrik would appreciate you doing a functional review of this PR.

**Review by ivs-cetmix — APPROVED** [review]
Code and functional review LGTM. 
Please update PR and commit messages with the changes context.
I would suggest using "Before/after" pattern for that.

**Comment by CetmixGitDrone** [comment]
This PR has the `approved` label and has been created more than 5 days ago. It should therefore be ready to merge by a maintainer (or a PSC member if the concerned addon has no declared maintainer). 🤖

File: cetmix_tower_server/demo/demo_data.xml [hunk 1]
```diff
@@ -243,11 +251,12 @@
...
         <field name="action">python_code</field>
         <field name="code">
-if {{ branch }}:
-    COMMAND_RESULT={"exit_code": 0, "message": "Branch is defined!"}
-else:
-    COMMAND_RESULT={"exit_code": -1, "message": "Branch is not defined!"}
+            if {{ branch }}:
+            COMMAND_RESULT={"exit_code": 0, "message": "Branch is defined!"}
+            else:
+            COMMAND_RESULT={"exit_code": -1, "message": "Branch is not defined!"}
```
Comment (GabbasovDinar) [inline cetmix_tower_server/demo/demo_data.xml]: why are these changes necessary here? also your changes is not python format 
please do not change the code formatting unless it is a bug, you are making an unnecessary update to git history.

File: cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py [hunk 2]
```diff
@@ -153,7 +152,11 @@ def execute_command_on_server(self):
         custom_values = {"log": {"label": log_label}}
         for server in self.server_ids:
             server.execute_command(
-                self.command_id, sudo=self.use_sudo, path=self.path, **custom_values
+                self.command_id,
+                sudo=self.use_sudo,
+                path=self.env.user.has_group("cetmix_tower_server.group_manager")
```
Comment (GabbasovDinar) [inline cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py]: `path=self.env.user ......` you can take it out of the loop so that you don't have to calculate this data every time.

**Review by GabbasovDinar — CHANGES_REQUESTED** [review]

File: cetmix_tower_server/demo/demo_data.xml (hunk 1 above)
Comment (Aldeigja) [reply cetmix_tower_server/demo/demo_data.xml]: I think it's some formatting done by pre-commit, I'll fix it.

**Review by GabbasovDinar — APPROVED** [review]
LGTM

**Comment by CetmixGitDrone** [comment]
This PR has the `approved` label and has been created more than 5 days ago. It should therefore be ready to merge by a maintainer (or a PSC member if the concerned addon has no declared maintainer). 🤖

**Comment by CetmixGitDrone** [comment]
Hey, thanks for contributing! Proceeding to merge this for you.
Prepared branch [14.0-dev-ocabot-merge-pr-94-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-94-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at 3a0bb2d394cd79cf6ddc1dcdaace279955c43591. Thanks a lot for contributing to cetmix. ❤️
